plugins/generator-1.19.2/forge-1.19.2/templates/rangeditem/projectile_entity.java.ftl

```diff
@@ -180,6 +184,12 @@ public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 		return entityarrow;
 	}
 
+	@Override
+	protected boolean tryPickup(Player entity) {

```

Comment:
This completely ignores the pickup status of projectiles shot in creative mode or with procedures. This method should be overridden only if the ammo item is empty, and you should only replace the `ALLOWED` case from super